base/rules/best-practices.js

```diff
@@ -215,6 +215,22 @@ module.exports = {
     // http://eslint.org/docs/rules/vars-on-top
     'vars-on-top': 2,
 
+    'valid-jsdoc': [2, {
+      prefer: {
+        'return': 'returns',
+      },
+      preferType: {
+        'object': 'Object',
+        'number': 'number',
+        'string': 'string',
+        'function': 'Function',
+        'array': 'Array'
+      },
```

Comment:
Why are some capitalized and others not? I would suggest we just go with this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures